src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  isSelected: boolean;
+  onSelect: () => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo: { id, completed, title },
+  isSelected,
+  onSelect,
+}) => {
+  return (
+    <tr
```

Comment:
don't do that, it's hard to read
```suggestion
export const TodoItem: React.FC<Props> = ({
  todo,
  isSelected,
  onSelect,
}) => {
  const { id, completed, title } = todo;
  
  return (
    <tr
    
```